### PR TITLE
Rewrite in Go: Pack Docker

### PR DESCRIPTION
**Pull request by dokshina**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,6 +30,7 @@ pytest:
 unit:
 	go test -v ./src/rpm
```
Comment (olegrok) [inline Makefile:31]: Are there something like "run all"?

May be simple `go test -v`?

File: src/project/project.go [hunk 2]
```diff
@@ -154,3 +159,15 @@ func detectName(path string) (string, error) {
 
 	return name, nil
 }
+
+func GetAppDir(projectCtx *ProjectCtx) string {
+	return filepath.Join("/usr/share/tarantool", projectCtx.Name)
```
Comment (olegrok) [inline src/project/project.go:164]: nit: seems it's a bit inconsistent `<path>/` and `<path>`.

I suggest you to add slash at the end of `/usr/share/tarantool` or drop for paths below.

File: test/python/test_pack.py [hunk 3]
```diff
@@ -555,22 +554,6 @@ def test_project_without_build_dockerfile(cartridge_cmd, project_without_depende
     assert process.returncode == 0
 
 
-@pytest.mark.skip()
```
Comment (olegrok) [inline test/python/test_pack.py:558]: Why do you drop tests?

Comment (dokshina) [reply test/python/test_pack.py:558]: This test duplicates the same one from `test_pack_docker`

File: Makefile (hunk 1 above)
Comment (dokshina) [reply Makefile]: Yes, changed to `go test -v ./src/...`.

**Pull request merged**